src/os/bluestore/AvlAllocator.cc

```diff
@@ -36,8 +36,8 @@ uint64_t AvlAllocator::_block_picker(const Tree& t,
 				     uint64_t align)
 {
   const auto compare = t.key_comp();
-  for (auto rs = t.lower_bound(range_t{*cursor, size}, compare);
-       rs != t.end(); ++rs) {
+  auto rs_last = t.lower_bound(range_t{*cursor, size}, compare);
```

Comment:
```suggestion
  const auto rs_start = t.lower_bound(range_t{*cursor, size}, compare);
```

i am not sure why we shall name it "last" though.

Reply:
Ok, can i name it "start_rs"?

Reply:
> i am not sure why we shall name it "last" though.

Ok, may i name it "start_rs"?